src/test/java/Homework19.java

```diff
@@ -0,0 +1,15 @@
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+    String deletemsg = "Deleted playlist";
+
+    @Test
+    public void deleteAPlaylist() throws InterruptedException {
+        logIn("heavenmayhem@gmail.com", "Everything2Me0628!");
+        choosePlaylist();
+        deleteButton();
+       // Assert.assertTrue(confirmDelete().contains(deletemsg));
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}

```

One Test - One Assert